doc/west/index.rst

```diff
@@ -0,0 +1,94 @@
+.. _west:
+
+West (Experimental)
+###################
+
+The Zephyr project is developing a swiss-army knife command line tool
+named ``west``. (Zephyr is an English name for the Greek god of the
+west wind.)
+
+West is developed in its own `repository on GitHub`_. A copy is
+currently maintained in Zephyr's :file:`scripts/meta/west` directory.
+
+Usage
+*****
+
+West's usage is similar to Git's: general options are followed by a
+command, which may also take options and arguments::
+
+  $ west [common-opts] <command-name> [command-opts] [<command-args>]
+
+Usage guides for specific groups of subcommands are in the following
+pages.
+
+.. toctree::
+
+   flash-debug.rst
+
+(This list will expand as additional features are developed.)
+
+Planned Features
+****************
+
+Additional features are under development for future versions of
+Zephyr:
+
+- Building Zephyr images.
+
+- Running Zephyr in emulation.
+
+- Bootloader integration: image building and signing, flashing
+  distinct images to different locations (the application is flashed
+  to separate area than the bootloader), and building and flashing the
```

Comment:
"a separate area"?

"the application and the bootloader are flashed to separate areas" maybe.

Reply:
IMO that would make the subsequent clause ("and building and flashing the bootloader itself") a bit redundant. But I agree that this is clumsily written. I'll try to streamline it.

Reply:
Replaced with:

```
- Bootloader integration: bootloader-aware image building, signing,
  and flashing, as well as building and flashing the bootloader itself.
```